packages/upswyng-web/src/components/Map.tsx

```diff
@@ -132,42 +132,48 @@ const Map = ({ address, name, latitude, longitude }: Props) => {
   );
   const [directionsError, setDirectionsError] = useState<string | null>(null);
 
-  const addMapMarker = () => {
-    const { address1, address2, city, state, zip } = address;
-
-    const resourceMarker = new googleMaps.Marker({
-      map: googleMap,
-      title: name,
-      position: {
-        lat: latitude,
-        lng: longitude,
-      },
-    });
+  useEffect(() => {
+    const addMapMarker = () => {
```

Comment:
It can, but it would then be redefined every re-render and cause the `useEffect` to fire each re-render as well. I can move it into a [`useCallback`](https://github.com/CodeForBoulder/upswyng/pull/236/commits/aed5be92e9bfc5484b178c5f8a4083c43fa70f98) to solve it as well. I don't think there will be a noticeable difference but hey why not ¯\\_(ツ)_/¯

Reply:
Your knowledge of hooks now greatly exceeds mine. I will step out.

Reply:
I've been creating quite a few custom ones at work lately 😅 
![now I am the master](https://media.giphy.com/media/YCIJVm0kcIEak/giphy.gif)